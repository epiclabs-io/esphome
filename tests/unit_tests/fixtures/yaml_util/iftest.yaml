---
substitutions:
  pins: [14, 15, 33, 39]

esphome:
  name: test

binary_sensor: !for
  items: ${pins}
  var: pin
  repeat: !if
    condition: ${pin > 20}
    then:
      platform: gpio
      pin: ${pin}
    else:
      platform: gpio
      pin: ${pin+10}
      name: 'shifted gpio ${pin}'
