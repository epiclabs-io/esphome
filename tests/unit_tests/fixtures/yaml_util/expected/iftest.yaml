binary_sensor:
- name: shifted gpio 14
  pin: 24
  platform: gpio
- name: shifted gpio 15
  pin: 25
  platform: gpio
- pin: 33
  platform: gpio
- pin: 39
  platform: gpio
esphome:
  name: test
